example/index.html

```diff
@@ -5,137 +5,10 @@
     <title>Fabric: Vue</title>
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="icon" href="/favicon.ico" />
-    <style>
-      f-docs-template {
-        visibility: hidden;
-      }
-      f-docs-template:defined {
-        visibility: visible;
-      }
-    </style>
-    <link
-      href="https://assets.finn.no/pkg/@fabric-ds/css/v0/fabric.min.css"
-      rel="stylesheet"
-    />
```

Comment:
Why was all of the template stuff wiped out?

Reply:
Template stuff was nuked because it caused problems with resolving the attention element position - IIRC this won't be a problem in podlets and such that use WC because of how the app's JS will be loaded by the WC

Reply:
Is it only a temporary solution until we merge all docs into one? Seems like it would break the common template we use across all of the documentation sites atm.

Reply:
Yup, I'll leave the Vue docs without the common template until we get the merged docs up. I can make them slightly prettier in the new year at least, but hopefully it won't take long to get the new docs up and running!